gene/fromGtf.go

```diff
@@ -87,11 +95,9 @@ func GtfToGene(g *gtf.Gene, ref []*fasta.Fasta) *Gene {
 				for i = answer.startPos - (val.FiveUtr.End - 1); i < answer.startPos-(val.FiveUtr.Start-2); i++ {
 					answer.featureArray[i] = UtrFive
 				}
-			}
-
-			if val.ThreeUtr != nil {
-				for i = answer.startPos - (val.ThreeUtr.End - 1); i < answer.startPos-(val.ThreeUtr.Start-2); i++ {
-					answer.featureArray[i] = UtrThree
+				answer.cdnaSeq = append(answer.cdnaSeq, answer.genomeSeq[answer.startPos-(val.FiveUtr.End-1):answer.startPos-(val.FiveUtr.Start-2)]...)
+				if answer.utrFive.end < answer.startPos-(val.FiveUtr.Start-2) {
```

Comment:
Similar to above, I am having trouble with the "if"s dealing with UTRs.  Not saying it is a bug, just that I am having trouble following everything.  Maybe either more comments, or shorter functions (where the function names then act as comments for what is happening)?

Reply:
Everything here is working to get them relative to the cDNA. I will see what I can do to break up the functions.